src/main/java/com/frcteam364/CTREModuleState.java

```diff
@@ -0,0 +1,56 @@
+package com.frcteam364;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.kinematics.SwerveModuleState;
+
+public class CTREModuleState {
+
+  /**
+   * Minimize the change in heading the desired swerve module state would require by potentially
+   * reversing the direction the wheel spins. Customized from WPILib's version to include placing
+   * in appropriate scope for CTRE onboard control.
+   *
+   * @param desiredState The desired state.
+   * @param currentAngle The current module angle.
+   */
+  public static SwerveModuleState optimize(SwerveModuleState desiredState, Rotation2d currentAngle) {
```

Comment:
Why not have the class extend SwerveModuleState and override the existing optimize function? Just a question as I am not a familiar with SwerveModuleState's usage, I am ok if the answer is, "it just won't work that way because ..."

Reply:
That would work but it would make confusing the two functions far easier. `CTREModuleState.optimize` is the correct one and `SwerveModuleState.optimize` is the incorrect one currently, but if you extend and override, both the functions look identical and are doing almost the same thing.

Reply:
If it is incorrect, what is its purpose vs SwerveModuleState.optimize? Why have/use CTREModuleState.optimize?